src/Input/ChoiceParam.php

```diff
@@ -59,4 +57,28 @@ public function getQuestion(): Question
             $defaultValue
         );
     }
+
+    /**
+     * @param null|string|string[] $defaultValue
+     */
+    private function createDefaultPrompt($defaultValue): string
+    {
+        if (null === $defaultValue) {
+            return '';
+        }
+
+        if (is_array($defaultValue)) {
+            Assert::isList($defaultValue);
+            Assert::allScalar($defaultValue);
+            $defaultValue = implode(', ', array_map(
+                /** @param bool|int|float|string $value */
```

Comment:
Shouldn't be needed, once `allScalar()` is removed

Reply:
See my other comment on this. It's... not straight-forward, due to how symfony/console handles this scenario.